pkg/expr/func.go

```diff
@@ -177,17 +213,17 @@ func newOR() Func {
 	}
 }
 
-// Call selects first non empty argument
+// Call selects first non empty argument / non erroneous
 // may return nil
-func (f *orFunc) Call(args []interface{}) interface{} {
-	if args[0] == nil {
-		return args[1]
+func (f *orFunc) Call(attrs attribute.Bag, args []*Expression, fMap map[string]FuncBase) (interface{}, error) {
+	for _, arg := range args {
+		ret, _ := arg.Eval(attrs, fMap)
```

Comment:
I like the collapsed thing, when the return value is important; but the call gets lost.
This way the *call* is important.